src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -560,7 +563,21 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model(const std::shared_ptr<
                                                           const ov::AnyMap& properties) const {
     OV_ITT_SCOPED_TASK(itt::domains::NPUPlugin, "Plugin::compile_model");
     OV_ITT_TASK_CHAIN(PLUGIN_COMPILE_MODEL, itt::domains::NPUPlugin, "Plugin::compile_model", "merge_configs");
-    auto localConfig = merge_configs(_globalConfig, any_copy(properties));
+
+    // Before going any further: if
+    // ... 1 - NPUW mode is activated
+    // ... 2 - this request is NOT coming from NPUW,
+    // activate the NPUW path
+    auto localProperties = properties;
+    auto useNpuwKey = ov::intel_npu::use_npuw.name();
+    if (localProperties.count(useNpuwKey) &&
+        localProperties.at(useNpuwKey).as<bool>() == true) {
+            return std::make_shared<ov::npuw::CompiledModel>
+                (model->clone(), shared_from_this(),
+                 localProperties);
+    }
```

Comment:
Not important, but I'm not sure we need the `localProperties` copy for the non-NPUW scenario. I'm thinking:
```suggestion
    auto useNpuwKey = ov::intel_npu::use_npuw.name();
    if (properties.count(useNpuwKey) && properties.at(useNpuwKey).as<bool>() == true) {
        const auto localProperties = properties;
        return std::make_shared<ov::npuw::CompiledModel>(model->clone(), shared_from_this(), localProperties);
    }
```

Reply:
I don't know why this copy is needed at all to be honest

Reply:
Most likely a refactoring left-over, removed, so far it works